home-assistant-plugin/custom_components/light/xknx.py

```diff
@@ -197,42 +204,12 @@ def color_temp(self):
     @property
     def min_mireds(self):
         """Return the coldest color temp this light supports in mireds."""
-        if self.device.supports_color_temperature or \
-                self.device.supports_tunable_white:
-            kelvin = self.device.max_kelvin
-            return color_util.color_temperature_kelvin_to_mired(kelvin) \
-                if kelvin is not None else DEFAULT_MIN_MIREDS
-        return None
```

Comment:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/light/__init__.py doesn't return None when color temperature is not supported so I think we're good always returning our default.

Reply:
Yep this looks good!